entity-framework/core/what-is-new/ef-core-5.0/whatsnew.md

```diff
@@ -14,6 +14,202 @@ This page does not duplicate the [plan for EF Core 5.0](xref:core/what-is-new/ef
 
 We will add links from here to the official documentation as it is published.
 
+## Preview 8
+
+## Table-per-type (TPT) mapping
+
+By default, EF Core maps an inheritance hierarchy of .NET types to a single database table. This is known as table-per-hierarchy (TPH) mapping. EF Core 5.0 also allows mapping each .NET type in an inheritance hierarchy to a different database table; known as table-per-type (TPT) mapping.
+
+For example, consider this model with a mapped hierarchy:
+
+```c#
+public class Animal
+{
+    public int Id { get; set; }
+    public string Species { get; set; }
+}
+
+public class Pet : Animal
+{
+    public string Name { get; set; }
+}
+
+public class Cat : Pet
+{
+    public string EdcuationLevel { get; set; }
+}
+
+public class Dog : Pet
+{
+    public string FavoriteToy { get; set; }
+}
+```
+
+By default, EF Core will map this to a single table:
+
+```sql
+CREATE TABLE [Animals] (
+    [Id] int NOT NULL IDENTITY,
+    [Species] nvarchar(max) NULL,
+    [Discriminator] nvarchar(max) NOT NULL,
+    [Name] nvarchar(max) NULL,
+    [EdcuationLevel] nvarchar(max) NULL,
+    [FavoriteToy] nvarchar(max) NULL,
+    CONSTRAINT [PK_Animals] PRIMARY KEY ([Id])
+);
+```
+
+However, mapping each entity type to a different table will instead result in one table per type:
+
+```sql
+CREATE TABLE [Animals] (
+    [Id] int NOT NULL IDENTITY,
+    [Species] nvarchar(max) NULL,
+    CONSTRAINT [PK_Animals] PRIMARY KEY ([Id])
+);
+
+CREATE TABLE [Cats] (
+    [Id] int NOT NULL,
+    [EdcuationLevel] nvarchar(max) NULL,
+    CONSTRAINT [PK_Cats] PRIMARY KEY ([Id])
```

Comment:
If this is going in the blog post, we may want to preemptively include the foreign key constraints here, so we don't get an unnecessary tweetstorm.

Reply:
I would rather not put in the blog post things that aren't in preview 8. But given I was using the wrong build, this is maybe not what is in preview 8?

If do still want to included this and this is what's generated for preview 8, then we could indicate that this is the SQL we generate in preview 8, but that it is changing in RC1.

Reply:
Thoughts, @AndriySvyryd?